### PR TITLE
Print groups of count group by response

### PR DESCRIPTION
**Pull request by rodrigozhou**
## What was changed

Print the groups returned by Count API.

## Checklist

1. Closes 

2. How was this tested:

Launched Temporal Server, and called `temporal workflow count -q 'group by ExecutionStatus'`

3. Any docs updates needed?

### CODE REVIEW

File: taskqueue/task_queue_commands.go [hunk 1]
```diff
@@ -265,12 +265,12 @@ func updateBuildIDs(c *cli.Context, partialReq workflowservice.UpdateWorkerBuild
...
 	}
 
-	resp, err := frontendClient.UpdateWorkerBuildIdCompatibility(ctx, request)
+	_, err = frontendClient.UpdateWorkerBuildIdCompatibility(ctx, request)
 	if err != nil {
 		return fmt.Errorf("error updating task queue build ids: %w", err)
 	}
 
-	fmt.Println(color.Green(c, "Successfully updated task queue build ids. Set ID: %v", resp.GetVersionSetId()))
+	fmt.Println(color.Green(c, "Successfully updated task queue build ids."))
```
Comment (alexshtin) [inline taskqueue/task_queue_commands.go:273]: Why did you remove this?

**Review by alexshtin — APPROVED** [review]

File: taskqueue/task_queue_commands.go (hunk 1 above)
Comment (cretz) [reply taskqueue/task_queue_commands.go:273]: This was already done/merged in #329
